running locally/Docker/readme.md

```diff
@@ -4,7 +4,7 @@
 
 This folder contains example files and steps for setting up Presto using Docker.
 <br>
-Soon we will have a video walk-through as well as a written walk through in blog form.
+Soon video walk-through will available as well as a written walk through in blog form.
```

Comment:
Suggest deleting this line. Soon is vague, and is a promise that might not be delivered on.

In general, technical documentation should use the present tense, and describe the current state of the software as well as the documentation.

Reply:
Okay. I deleted the comment.

Reply:
Thank you!